stubs/MockBuilder.stub

```diff
@@ -26,4 +26,14 @@ class MockBuilder
 	 */
 	public function getMockForAbstractClass() {}
 
+	/**
+     * @phpstan-return MockBuilder
+     */
+    public function setMethods(array $methods = null) {}
```

Comment:
@ondrejmirtes I thought this would have fixed https://github.com/phpstan/phpstan-phpunit/issues/87

I would have expect that the MockBuilder will lose his template param
```
$this->getMockBuilder(Bar::class)
```
would be MockBuilder<Bar> so
```
$this->getMockBuilder(Bar::class)->getMock()
```
would be MockObject&Bar but I wanted that
```
$this->getMockBuilder(Bar::class)->setMethods(...)
```
become something like MockBuilder<mixed> so
```
$this->getMockBuilder(Bar::class)->setMethods(...)->getMock()
```
become MockObject&mixed and every method are allowed

This could be maybe a first step for solving the issue.
But it seems like I'm doing this wrong...